src/inc/clrnt.h

```diff
@@ -837,6 +837,13 @@ RtlVirtualUnwind_Unsafe(
 
 #ifdef _TARGET_X86_
 #ifndef FEATURE_PAL
+//
+// x86 ABI does not define RUNTIME_FUNCTION. Define our own to allow unification between x86 and other platforms.
```

Comment:
It turns out that windows build uses PORTABILITY_ASSERT that raises compile error (instead of runtime error).